Dockerfile

```diff
@@ -1,22 +1,20 @@
 # syntax=docker/dockerfile-upstream:experimental
 
-FROM phusion/baseimage:0.11
+FROM ubuntu:18.04
 
 RUN apt-get update -qq && apt-get install -y \
     git \
     cmake \
     g++ \
-    protobuf-compiler \
     pkg-config \
     libssl-dev \
-    unzip \
-    systemd-coredump \
```

Comment:
Why is this removed?

Reply:
unzip: doesn't needed. systemd-coredump: will mess the coredump location and even we want that, it should be installed in host since coredump is kernel level, docker shares kernel with host